.github/workflows/python-package-conda.yml

```diff
@@ -0,0 +1,34 @@
+name: Python Package using Conda
+
+on: [push]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 5
```

Comment:
Shouldn't this be a matrix with at least the Python versions supported on conda?